SUG.py

```
@@ -0,0 +1,129 @@
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Flake8 (7.2.0)

[error] 'from website import *' used; unable to detect undefined names

(F403)